mapbox/services/uploads.py

```diff
@@ -1,7 +1,7 @@
 from boto3.session import Session as boto3_session
 from uritemplate import URITemplate
 
-from mapbox.errors import InvalidFileError
+from mapbox.errors import InvalidFileError, ValidationError
```

Comment:
Put `from mapbox.errors import ValidationError` on a separate line to avoid a conflict with the change in made in #158 (removing the import of `InvalidFileError`).